release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -156,6 +156,7 @@ With this release, there are several updates to the *Administrator* perspective
 
 * The {product-title} web console displays a `ConsoleNotification` if the cluster is upgrading. Once the upgrade is done, the notification is removed.
 * A *_restart rollout_* option for the `Deployment` resource and a *_retry rollouts_* option for the `DeploymentConfig` resource are available on the *Action* and *Kebab* menus.
+* You can view a list of supported cluster products on the *_Cluster list_* dropdown menu on the the *{product-title} Cluster Manager* page. The supported products include {product-title}, {product-title} (ROSA), Azure Red Hat OpenShift (ARO), ROKS, and {product-dedicated}. 
```

Comment:
The supported cluster list is : {"OpenShift", "ROSA", "ARO", "ROKS", "OpenShiftDedicated"} while `ROSA` is short for `[Red Hat OpenShift Service on AWS (ROSA)](https://cloud.redhat.com/products/amazon-openshift)`

Reply:
Also in 4.12, the main update is to show the supported clusters in `All Clusters` dropdown in OCP console not `Cluster list dropdown menu on the the OpenShift Container Platform Cluster Manager page`

Reply:
<img width="241" alt="Screenshot 2023-01-09 at 3 45 35 PM" src="https://user-images.githubusercontent.com/12692381/211260875-c17c18c5-047c-4f06-8a20-a940d711922d.png">